notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1916 @@
+{
```

Comment:
<p>Not important: another link for TruncatedDistribution: https://num.pyro.ai/en/stable/_modules/numpyro/distributions/truncated.html#TruncatedDistribution</p><p> </p><p>Typo: Predictive (and its link: https://num.pyro.ai/en/stable/utilities.html#predictive)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>